nav/stepper/stepper_constants.py

```diff
@@ -1,26 +1,32 @@
+import math
 # Stepper mode select
 STEPPER_m0 = 17
 STEPPER_m1 = 27
 STEPPER_m2 = 22
 
 # First wheel GPIO Pin
-WHEEL1_step = 21
-WHEEL1_dir = 20
+BR_step = 21
+BR_dir = 20
 
 # Second wheel GPIO Pin
-WHEEL2_step = 11
-WHEEL2_dir = 10
+FR_step = 11
+FR_dir = 10
 
 # Third wheel GPIO Pin
-WHEEL3_step = 7
-WHEEL3_dir = 8
+BL_step = 7
+BL_dir = 8
 
 # Fourth wheel GPIO Pin
-WHEEL4_step = 24
-WHEEL4_dir = 23
+FL_step = 24
+FL_dir = 23
+
+#Direction constants
+FWD = True
```

Comment:
Talk to @BenjiLee0002 about this. All main constants should be in the same place for accessibility.